web/pom.xml

```diff
@@ -721,6 +721,54 @@
             </fileset>
           </filesets>
         </configuration>
+        <executions>
+          <execution>
+            <id>reset</id>
+            <goals><goal>clean</goal></goals>
+            <configuration>
+              <filesets>
+                <fileset> <!-- database -->
+                  <directory>.</directory>
+                  <includes>
+                    <include>gn*</include>
+                  </includes>
+                </fileset>
+                <fileset>
+                  <directory>jcs_caching</directory>
+                  <includes>
+                    <include>**</include>
+                  </includes>
+                </fileset>
+                <fileset>
+                  <directory>logs</directory>
+                  <includes>
+                    <include>**</include>
+                  </includes>
+                </fileset>
+                <fileset>
+                  <directory>images</directory>
+                  <includes>
+                    <include>**</include>
+                  </includes>
+                </fileset>
+              </filesets>
+            </configuration>
+          </execution>
+          <execution>
+            <id>js</id>
+            <goals><goal>clean</goal></goals>
+            <configuration>
+              <filesets>
+                <fileset>
+                  <directory>jcs_caching</directory>
+                  <includes>
+                    <include>**</include>
+                  </includes>
+                </fileset>
+              </filesets>
```

Comment:
I think `web/jcs_caching` isn't the JS cache. JS cache is a H2 database stored in the GeoNetwork data folder with the name `wro4j-cache*`. 
The files to remove would be `src/main/webapp/WEB-INF/data/wro4j-cache*`.

Reply:
I see the `jcs_caching` folder created, what is this for?

Reply:
Thanks, I see https://commons.apache.org/proper/commons-jcs/index.html